src/index.html

```diff
@@ -11,8 +12,55 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Browsers cannot directly interpret SCSS files. Ensure that the stylesheet link points to a compiled CSS file instead of an SCSS file.